src/fidesops/schemas/saas/saas_config.py

```diff
@@ -63,19 +63,36 @@ class Strategy(BaseModel):
     configuration: Dict[str, Any]
 
 
+class HTTPMethod(Enum):
+    """Enum to represent HTTP Methods"""
+
+    GET = "GET"
+    POST = "POST"
+    PATCH = "PATCH"
+    PUT = "PUT"
+    DELETE = "DELETE"
```

Comment:
Should we add `OPTIONS` and `HEAD` here too? I can see those being useful for further connectors.